docs/run/_toc.json

```
@@ -41,7 +41,7 @@
       ]
     },
     {
-      "title": "Sessions",
+      "title": "Scheduling modes",
```

```suggestion
      "title": "Execution modes",
```

`Execution modes` is the term we are presenting at summit for batch + session.